README.md

```diff
@@ -243,4 +241,41 @@ On MacOS or Windows systems, we recommend either using a VM or the provided `.de
 - `build`: builds and tags `envbuilder:latest` for your current architecture.
 - `develop`: runs `envbuilder:latest` against a sample Git repository.
 - `test`: run tests.
-- `test-registry`: stands up a local registry for caching images used in tests.
\ No newline at end of file
+- `test-registry`: stands up a local registry for caching images used in tests.
+
+<!--- PLEASE DO NOT MODIFY THIS SECTION. IT IS AUTOMATICALLY GENERATED. --->
+<!--- start-flags --->
+## Flags
```

Comment:
```suggestion
## CLI Flags
```

Reply:
I would suggest renaming this "Environment Variables" for clarity.